test/integration/shoots/logging/utils.go

```diff
@@ -34,8 +35,8 @@ func hasRequiredResources(ctx context.Context, k8sSeedClient kubernetes.Interfac
 		return false, err
 	}
 
-	fluentd := &appsv1.StatefulSet{}
-	if err := k8sSeedClient.DirectClient().Get(ctx, client.ObjectKey{Namespace: garden, Name: fluentdName}, fluentd); err != nil {
+	loki := &appsv1.StatefulSet{}
+	if err := k8sSeedClient.Client().Get(ctx, client.ObjectKey{Namespace: garden, Name: lokiName}, loki); err != nil {
```

Comment:
you should use the direct client there as the `Client` is not initialized for the test framework afaik @tim-ebert

Reply:
Yes, the cached client is not started in the integration tests, so you should use `DirectClient`.